app/assets/javascripts/editorToolbar.js

```diff
@@ -1,29 +1,9 @@
-// this script is used in a variety of different contexts including:
+// this script is used wherever the legacy editor is used.
 //   pages (wikis, questions, research notes) with multiple comments & editors for each comment
 //   pages with JUST ONE form, and no other comments, eg. /wiki/new & /wiki/edit
 //   /app/views/features/_form.html.erb
 //   /app/views/map/edit.html.erb
-//   the legacy editor: /app/views/editor/_editor.html.erb (if it's still in use live?)
-
-const getEditorParams = (targetDiv) => {
-  const closestCommentFormWrapper = targetDiv.closest('div.comment-form-wrapper'); // this returns null if there is no match
```

Comment:
This legacy code is not at all good, using `ids` everywhere is so awesome 💯

Reply:
I know right!!!